installer/charts/rhtap-acs-scanner/templates/service-account.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Does it need "write" permission to Secrets?

Reply:
In fact, we don't need the SA at all.
I now use the `default` SA in the `rhtap` namespace.